src/navigation/DrawerNavigator.tsx

```diff
@@ -96,20 +95,15 @@ export const DrawerNavigator = () => {
         />
       )}
       screenOptions={{
-        drawerPosition:"right",
-        headerShown: false
+        drawerPosition: 'right',
+        headerShown: false,
+        drawerStyle: {
+          width: device.width > device.height ? device.height * 0.8 : device.width * 0.8
+        }
```

Comment:
Why did you incorporate these styles into the component? Is there a special reason? Because I can't tell from the commit message.

Reply:
These additional styles were needed to keep the before used design with the new updates

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/901 in the drawer now bottom picture in this pr its visible that the drawer width is not as wide as it was before

Reply:
I think the question I want to ask is not understood. Could this style be used as follows?

```
screenOptions={{
  drawerPosition: 'right',
  headerShown: false,
  drawerStyle: styles.drawerStyle
}}
...
const styles = StyleSheet.create({
  drawerStyle: {
    width: device.width > device.height ? device.height * 0.8 : device.width * 0.8
  }
});
```

Reply:
yes it can be used but I figured that the other solution is better since we don't use any other styles in this file, therefore we can save ourselves 4 additional lines of code which are practically not needed

Reply:
i understand and think we can keep it in the render part.
but, i will sort props alphabetically and try to user our orientation provider to determine landscape/portrait instead of width height comparison here

Reply:
9d52c48